packages/suite-web/e2e/tests/wallet/send-form-doge.test.ts

```diff
@@ -0,0 +1,65 @@
+// @group:wallet
+// @retry=2
+
+describe('Doge send form with mocked blockbook', () => {
+    beforeEach(() => {
```

Comment:
Duplicated code from previous commit, could be an imported function. Same for `afterEach`.